modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -66,158 +125,332 @@ The following table compares the Object Store REST API operations to Object Stor
+  -H "Authorization: Bearer ACCESS_TOKEN" \
+  -H "Content-Type: application/json" \
+  --data '{"stringValue":"Hello World","keyId":"myTestKey","valueType":"STRING"}'
 ----
 
-Substitute CLIENT_ID_VALUE and CLIENT_SECRET_VALUE for the Client ID and secret.
+In this command, the value is a JSON string. 
 
-The output appears as:
+
+To verify that the key-pair is in the object store, get a list of the keys in the partition:
+
+[source,console,linenums]
+----
+curl -X GET \
+  https://BASE_URL/api/v1/organizations/ORG_ID/environments/ENV_ID/stores/STORE_ID/partitions/PARTITION_ID/keys \
+  -H "Authorization: Bearer ACCESS_TOKEN" 
+----
+
+This command returns output like the following:
 
 [source,json,linenums]
 ----
-{
-    "access_token": "42424242-5454-4242-5454-425442544254",
-    "token_type": "bearer"
-}
+{"values":[{"keyId":"KeyTwo"},{"keyId":"KeyOne"},{"keyId":"myTestKey"}],"nextPageToken":null}
 ----
 
-== Example: Retrieve a List of Object Stores
 
-The following procedures assume you have already created an application and given it access to Object Store v2.
+==== Example: Delete a Key and Value 
+
+To delete a key-value pair, submit a `curl` command like the following, 
+replacing BASE_URL with the regional domain, BEARER_TOKEN with your bearer token, 
+ORG_ID with your organization ID, 
+ENV_ID with your environment ID, the STORE_ID with the name of the object store, and the PARTITION_ID with the name of the partition, and the KEY_ID with the key ID:
+
+[source,console,linenums]
+----
+curl -X DELETE \
+  https://BASE_URL/api/v1/organizations/ORG_ID/environments/ENV_ID/stores/STORE_ID/partitions/PARTITION_ID/keys/KEY_ID \
+  -H "Authorization: Bearer ACCESS_TOKEN"
+----
+
+In this command, the value is a JSON string. 
 
-*Note:* Get the Client ID and Secret as described in <<View the App Client ID and Secret>>.
 
-=== Get Environment and Organization IDs
+To verify that the key-pair is no longer in the object store, get a list of the keys in the partition:
 
-In *Access Management > Environments*, click the environment where the application resides. Copy
-the environment ID from the URL, for example:
+[source,console,linenums]
+----
+curl -X GET \
+  https://BASE_URL/api/v1/organizations/ORG_ID/environments/ENV_ID/stores/STORE_ID/partitions/PARTITION_ID/keys \
+  -H "Authorization: Bearer ACCESS_TOKEN" 
+----
 
-`+https://anypoint.mulesoft.com/accounts/#/cs/core/environments/edit/badd09f00f004242badd0942+`
+This command returns output like the following:
 
-In *Access Management > Organization*, copy your *Organization ID*.
+[source,json,linenums]
+----
+{"values":[{"keyId":"KeyTwo"},{"keyId":"KeyOne"}],"nextPageToken":null}
+----
 
-=== View List of Object Stores
 
-Submit this command replacing `BEARER_ID` with your bearer ID, `ORG_ID` with your organization ID, and `ENV_ID` with your environment ID.
+
+== Object Store Stats API 
+
+The Object Store Stats API enables you to retrieve statistics on your Object Store usage.
```

Comment:
ditto prev. comments about the massive amount of different types of info in this set, when the two points are one API enabling you to temp. store and retrieve Object Store key-value pairs and the other to retrieve Object Store usage stats

Reply:
The follows the MQ API docs. If I break it into two, it overemphasizes the importance of the API, which I don't want to do.